jax/_src/ops/scatter.py

```diff
@@ -178,6 +178,8 @@ def index_add(x: Array,
                [1., 1., 1., 7., 7., 7.],
                [1., 1., 1., 1., 1., 1.]], dtype=float32)
   """
+  warnings.warn("index_add is deprecated. Use x.at[idx].add(y) instead.",
+                DeprecationWarning)
```

Comment:
I would lean toward using `FutureWarning` instead, because I believe `DeprecationWarning` is ignored by default unless the function is called directly in `__main__`.

Reply:
OK, that makes sense.